BlueprintUI/Sources/Element/UIViewElement.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  UIViewElement.swift
+//  BlueprintUI
+//
+//  Created by Kyle Van Essen on 6/4/20.
+//
+
+import UIKit
+
+
+///
+/// An element type  which makes it easier to wrap an existing `UIView` instance that
+/// provides its own sizing via `sizeThatFits`. An instance of the view is used for
+/// sizing and measurement, so that you do not need to re-implement your own measurement.
+///
+/// Example
+/// -------
+/// If you were implementing a very basic `Image` element, your implementation would look something
```

Comment:
`Image` is also kind of a bad example, because it's easy to measure the size of an image without creating a UIImageView.

I think we should strive to make the docs clear that this is not the first tool you should reach for when building a new element. If you can measure the size without a view instance, that is preferable. This helps in the cases where that's impossible (because you don't own the view and the measurement is a black box) or cumbersome (because it's an existing composite view with complex logic you don't want to extract).

Maybe something like a fictitious `LegacyBarGraphView` for an example?

Reply:
Every example has a positive and negative. I chose this and previous example of label because it's pretty immediately obvious to the reader what the model and view backing it is

Reply:
I think UISwitch is a good example to use here; gonna switch to that, since I've run into it before